antarest/study/storage/variantstudy/model/command/create_xpansion_matrix.py

```diff
@@ -90,10 +80,6 @@ class CreateXpansionCapacity(AbstractCreateXpansionMatrix):
 
     command_name: CommandName = CommandName.CREATE_XPANSION_CAPACITY
 
-    @override
-    def _apply_config(self, study_data: FileStudyTreeConfig) -> Tuple[CommandOutput, Dict[str, Any]]:
-        return apply_config_create_resource_commands(self.filename, XpansionResourceFileType.CAPACITIES)
```

Comment:
The function `apply_config_create_resource_commands` can now be removed and `apply_create_resource_commands` simplified.